interface-playground.ipynb

```diff
@@ -412,21 +429,17 @@
    "outputs": [],
    "source": [
     "result = main(\n",
-    "    inputs={\n",
-    "        \"date_str\": \"2025-01-01\",\n",
-    "        \"input_data\": {\n",
-    "            \"df_and_mapper\": {\n",
-    "                \"df\": DATA,\n",
-    "                \"mapper\": TREE_TO_DF_MAPPER,\n",
-    "            },\n",
-    "        },\n",
-    "        \"targets\": {\n",
-    "            \"tree\": TARGETS_TREE,\n",
-    "        },\n",
-    "        \"policy_environment\": policy_environment,\n",
-    "    },\n",
-    "    output_names=[\"results__df_with_mapper\"],\n",
-    ")[\"results__df_with_mapper\"]\n",
+    "    date_str=\"2025-01-01\",\n",
+    "    policy_environment=modified_policy_environment,\n",
```

Comment:
are all nodes of the interface-dag potential arguments of the `main` function?

Is there an easy way to know which arguments I need to compute which `main_targets`?